lib/cypress_on_rails/configuration.rb

```diff
@@ -2,8 +2,9 @@
 
 module CypressOnRails
   class Configuration
-    attr_accessor :cypress_folder
     attr_accessor :api_prefix
+    attr_accessor :install_folder
+    attr_accessor :cypress_folder
```

Comment:
I think we rather create a fallback wrapper instead.

```suggestion
  def cypress_folder
    warn "cypress_folder is deprecated, please use install_folder"
    install_folder
  end

  def cypress_folder=(v)
    warn "cypress_folder= is deprecated, please use install_folder"
    self.install_folder = v
  end
```

Reply:
Ping @khaledEmaraDev